harbor-ui/Cargo.toml

```diff
@@ -8,25 +8,20 @@ default = []
 vendored = ["harbor-client/vendored"]
 
 [dependencies]
+anyhow = { workspace = true }
```

Comment:
goal was to keep anyhow out of harbor-ui not sure when it slipped in but I'll make an issue for it #223